ecs-terraform/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM hashicorp/terraform:1.9.7 AS base
+
+RUN mkdir -p /app
+WORKDIR /app
+ENTRYPOINT [ "sh", "-c" ]
+
+COPY --from=top-level-directory bin /app/bin
+COPY --from=top-level-directory infra /app/infra
+COPY --from=top-level-directory Makefile /app/Makefile
```

Comment:
Fancy new docker features here, ability to pull from a context above your current folder